src/Reveal.Sdk.Dom.Tests/Visualizations/OHLCVisualizationFixture.cs

```diff
@@ -0,0 +1,361 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class OHLCVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var ohlcVisualization = new OHLCVisualization();
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
```

Comment:
Remove this, b/c the codes below already ensure that ohlcVisualization is not null